src/Laravel/WebHookArtisanCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Telegram\Bot\Laravel;
+
+use Illuminate\Console\GeneratorCommand as LaravelGeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+
+class WebHookArtisanCommand extends LaravelGeneratorCommand
+{
+    /**
+     * Create a new command instance.
+     *
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
```

Comment:
Do we need the constructor for this command? I think you can omit it for now.

Reply:
For injecting other classes. Like Api class i think...